lib/term.rb

```diff
@@ -0,0 +1,17 @@
+require 'nokogiri'
+require_relative 'table'
+
+class Term < Page
+
+  field :members do
+    tables.map do |table|
+      Table.new(table).to_a
+    end.flatten
```

Comment:
`.map { … }.flatten` is usually cleaner as `.flat_map`

(and would let you collapse this into `tables.flat_map { |node| Table.new(node).to_a }`